chromium_src/chrome/browser/sessions/session_common_utils.cc

```diff
@@ -3,7 +3,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-#include "brave/common/extensions/extension_constants.h"
+#include "extensions/common/constants.h"
```

Comment:
this doesn't appear to be used, although it looks like it should have `ethereum_remote_client_extension_id` below, but both the header and the extension host check should be guarded by a buildflag

Reply:
Addressed by 6ed57b43acf10442bb0a31f395b84c0ed3831ffb